navansatt/src/main/kotlin/no/nav/amt/navansatt/NavAnsattRepository.kt

```diff
@@ -23,38 +21,47 @@ open class NavAnsattRepository(
 		)
 	}
 
-	fun upsert(upsertCmd: UpsertNavAnsattCommand) {
+	internal fun upsert(upsertCmd: NavAnsattDbo) {
```

Comment:
Personlig så foretrekker jeg Command eller hva vi enn skal kalle det istedenfor Dbo, siden det gjør det mye mer tydelig hva som faktisk insertes.

Reply:
Vi brukte ikke Command noen andre steder såvidt jeg kunne se, så tenkte det var greit å være konsistente. Eller burde vi vurdere å innføre command alle steder vi inserter?

Reply:
Vi har det kanskje bare for test-modulen pr nå. Vi kan godt endre det i denne PRen, men føler fortsatt at det ofte er cleanere å ha egne data klasser som beskriver hva som insertes/upsertes.

Reply:
Ta tar jeg å merger den og lenker for mer prat i #team_komet_utviklere

Reply:
Ja til endre alle steder eller la være eller så blir det bare søl